.github/workflows/docs-needed.yml

```
@@ -0,0 +1,441 @@
+          SOURCE_REPO: ${{ steps.vars.outputs.source_repo }}
+          PR_NUMBER:   ${{ steps.vars.outputs.pr_number }}
+          PR_TITLE:    ${{ steps.vars.outputs.pr_title }}
+          PR_AUTHOR:   ${{ steps.vars.outputs.pr_author }}
+        run: |
+          mkdir -p "$(dirname "$DOCS_FILE")"
+ 
+          # Overwrite the file on every run so a re-trigger regenerates cleanly
+          cat > "$DOCS_FILE" <<HEADER
+# ${PR_TITLE}
+ 
+> **Status:** Draft — auto-generated. Requires editorial review before publishing.
```

actionlint (1.7.12)

[error] could not parse as YAML: did not find expected comment or line break

(syntax-check)